msteams-platform/TOC.yml

```diff
@@ -98,19 +98,19 @@
     items:
     - name: Teams Toolkit overview
       href: toolkit/teams-toolkit-fundamentals.md
-    - name: Create a new Teams app
+    - name: Create a new Teams app using Teams Toolkit
       href: toolkit/create-new-project.md
     - name: Prepare accounts to build Teams app
       href: toolkit/accounts.md
     - name: Debug your Teams app locally
       href: toolkit/debug-local.md
-    - name: Debug background processes
+    - name: Debug background process
       href: toolkit/debug-background-process.md
-    - name: Provision cloud resources
+    - name: Use Teams Toolkit to provision cloud resources
       href: toolkit/provision.md
     - name: Deploy Teams app to the cloud
       href: toolkit/deploy.md
-    - name: Preview and customize Teams app manifest
```

Comment:
I don't know it but is there a functionality to preview the manifest? If not, then let's remove the word preview. If yes, then definitely retain the word preview.

Reply:
It has been changed to Edit Teams app manifest.